app/src/main/kotlin/at/bitfire/davdroid/servicedetection/CollectionListRefresher.kt

```diff
@@ -56,6 +56,14 @@ class CollectionListRefresher @AssistedInject constructor(
     }
 
 
+    /**
+     * The HttpUrls which have been fetched already during a single refresh.
+     */
+    private val alreadyFetched = mutableSetOf<HttpUrl>()
+
+    /**
+     * The HttpUrls which have been queried already during a single refresh.
+     */
     private val alreadyQueried = mutableSetOf<HttpUrl>()
```

Comment:
What's the difference between "fetched" and "queried"?

Reply:
Queried addresses are addresses which we have sent a request / propfind to.
Fetched addresses are addresses which are saved/remembered/"downloaded" in order to make a request later.

Reply:
Should probably make that clear in the kdoc.

Reply:
> Should probably make that clear in the kdoc.

Definitely 😅